ChangeLog.md

```diff
@@ -24,6 +24,9 @@ See docs/process.md for more on how version tagging works.
 3.1.73 - 11/28/24
 -----------------
 - libunwind was updated to LLVM 19.1.4. (#22394)
+- mimalloc was updated to 2.1.7. (#21548)
```

Comment:
I wonder how this line ended up here?

Reply:
Yeah I managed to remove it in #23017 and add it here...